CHANGELOG.md

```diff
@@ -5,15 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 
-## [Unreleased, planned for 2024-04-30]
+## [Unreleased, planned for 2024-05-07]
```

Comment:
This is ok, but we should discuss later weather an "Unreleased" section is useful for us.

Reply:
This is the sections where PRs should add a short summary describing the change. Please note that usually this CHANGELOG will need kind of a "translation" for end users of the ipl plattform.